docs/13_upload.md

```diff
@@ -0,0 +1,127 @@
+# Uploading Data
+
+## Creating an Account on DANDI
+In order to create a new `Dandiset` and upload your data, you need to have a DANDI account.
+
+To create a DANDI account you need:
+
+1. [Create a GitHub account](https://github.com/) if you don't have one.
+2. Using your GitHub account, [register a DANDI account](https://gui.dandiarchive.org/#/user/register).
+
+You will receive an email acknowledging activation of your account within 24
+hours, after which you can log in to DANDI using GitHub by clicking the login
+button.
+
+## Creating Dandiset and adding the data
+
+DANDI provides a production server (http://dandiarchive.org/) for real data and an identical staging server 
+(https://gui-staging.dandiarchive.org/) for test data.
+When you create a Dandiset, a permanent ID is automatically assigned to it.
+To prevent the production server from being inundated with test Dandisets, we encourage developers to develop 
+against the staging server. The below instructions will alert you to where the commands for interacting with these 
```

Comment:
```suggestion
To prevent the production server from being inundated with test Dandisets, we encourage developers to develop 
against the development server (https://gui-staging.dandiarchive.org/). Please note that the development server
should not be used to stage your data. All data are uploaded as draft and can be adjusted before publishing on
the production server. The development server is primarily used by users learning to use DANDI or by developers.

The below instructions will alert you to where the commands for interacting with these 
```